src/helpers/getUniqueWord.ts

```diff
@@ -29,17 +31,17 @@ export const wordsArrayGenerator = (wordsQuantity: number) => {
       Math.random() * wordsDb.categories[categoryNumber].words.length
     );
 
-    const actualWord = wordsDb.categories[categoryNumber].words[wordNumber];
+    actualWord = wordsDb.categories[categoryNumber].words[wordNumber];
 
     // Security check, we skip composed words
     if (actualWord.split(' ').length > 1) continue;
 
     // Security check, we only add unique words
     if (wordsArray.includes(actualWord)) continue;
 
-    // If here, we store the lowered case word
-    wordsArray.push(actualWord.toLocaleLowerCase());
-  } while (wordsArray.length < wordsQuantity);
+    // If here, the word is unique, we can leave do while loop
+    newWordFound = true;
+  } while (newWordFound == false);
```

Comment:
the use of non-strict comparison is necessary here? (I mean is better everywhere use strict one in js)

Reply:
You are right, I am fixing this right now

Reply:
Done, PR updated